config.toml

```diff
@@ -36,8 +40,8 @@ fancy_code = true
 dynamic_note = false
 
 menu = [
-    { name = "/posts", url = "/posts", weight = 1 },
```

Comment:
It is necessary to remove the slashes due to the lang code being prefixing the URLs generated by the `get_url` calls. Without this change, there would have been two slashes at the start of URLs with the `de` lang code.